crates/swc_ecma_codegen/src/lib.rs

```diff
@@ -3070,7 +3071,12 @@ fn unescape_tpl_lit(s: &str, is_synthesized: bool) -> String {
     result
 }
 
-fn escape_without_source(v: &str, target: EsVersion, single_quote: bool) -> String {
+fn escape_without_source(
+    v: &str,
+    target: EsVersion,
+    single_quote: bool,
+    emit_non_ascii_as_unicode: bool,
```

Comment:
setting this to `true` is equivalent to disabling `jsesc` `minimal` option

https://github.com/mathiasbynens/jsesc#minimal